binaries/Cargo.toml

```diff
@@ -5,6 +5,7 @@ edition = "2018"
 description = "Prio utilities"
 license = "MPL-2.0"
 repository = "https://github.com/divviup/libprio-rs"
+resolver = "2"
```

Comment:
I think this may be extraneous because it's implied by the workspace's `resolver = "2"`. From https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html:

> The resolver is a global setting for a [workspace](https://doc.rust-lang.org/cargo/reference/workspaces.html), and the setting is ignored in dependencies. The setting is only honored for the top-level package of the workspace.